build_locally.sh

```diff
@@ -18,11 +28,22 @@ gem install jekyll
 gem install bundler -v 1.17.3
 bundle install
 
-# Build website using similar script as Travis, but skip htmlproofer because it fails locally:
-bundle exec jekyll build --strict_front_matter
+# Optionally test website using same script as Travis.
+# When you are developing locally, you may or may not want to run the full set of tests that Travis does
+if have_travis_arg "$@"; then
+  ./.travis.sh
+fi
 
 # Launch website
-echo "Opening http://localhost:4000 in default browser"
+echo
+echo "-------------------------------------"
+echo "-------------------------------------"
+echo "Preparing to open http://localhost:4000"
+echo "Be sure to refresh auto-opened webpage"
+read -p "Press any key to start serving website..."
```

Comment:
The additional information for refreshing is useful for first time editors but I think the additional step for everytime someone wants to test their edit outweighs the need for user interaction, I would want the read -p to be replaced with an echo command, but that feels more like a personal choice.

Reply:
i removed ``read -p``